css/style/card.css

```diff
@@ -0,0 +1,91 @@
+.card{
+   position: relative;
+    background-color: var( --bg-secondary);
+    border-radius: 10px;
+    display:inline-block;
+}
+.simple-card{
```

Comment:
Give at least a line space between two declarations. Else looks awesome. Nice work.

Just add a line space.

Reply:
Okay ! Thanks for reviewing :)